README.md

```diff
@@ -73,6 +103,15 @@ The generator installs your webpack files in the `client` folder. Foreman uses w
 
 Inside your Rails views, you can now use the `react_component` helper method provided by React on Rails.
 
+### Client-Side Rendering vs. Server-Side Rendering
+In most cases, you should use the `prerender: false` (default behavior) with the provided helper method to render the React component from your Rails views. In some cases, such as when SEO is vital or many users will not have JavaScript enabled, you can enable server-rendering by passing `prerender: true` to your helper, or you can simply change the default in `config/initializers/react_on_rails`. Your JavaScript can then be first rendered on the server and passed to the client as HTML.
+
+Note that **server-rendering requires globally exposing your components by setting them to `global`, not `window`** (as is the case with client-rendering). If using the generator, you can pass the `--server-rendering` option to configure your application for server-side rendering. 
+
+In the following screenshot you can see the actual HTML rendered for a side-by-side comparison of a React component left as JavaScript for the client to render followed by the same component rendered on the server to HTML along with any console error messages generated:
```

Comment:
In the following screenshot you can see the 3 parts of react_on_rails rendering:

1. Inline JavaScript to "client-side" render the React component.
2. The wrapper div `<div id="HelloWorld-react-component-0">` enclosing the server rendered HTML for the React component
3. Additional JavaScript to console log any messages, such as server rendering errors.

**Note**: If server rendering is not used (prerender: false), then the major difference is that the HTML rendered contains *only* the outer div: `<div id="HelloWorld-react-component-0"/>`